learn/index.md

```diff
@@ -212,5 +212,9 @@ title: Let’s learn Ballerina!
       <h3><a href="/learn/how-to-write-secure-ballerina-code">Writing Secure Ballerina Code</a></h3>
       <p>Provides guidelines on writing secure Ballerina services, while demonstrating different security features and controls available within Ballerina.</p>
    </div>
+   <div class="col-sm-12 col-md-6 cTwoCol">
+      <h3><a href="/learn/how-to-extend-ballerina">Extending Ballerina</a></h3>
+      <p>This guide shows how additional compile-time behavior can be added through compiler extensions and annotations.</p>
```

Comment:
Should we replace behavior with "analysis/validation".

Shall we also replace
```suggestion
      <p>This guide shows how additional compile-time behavior can be added using compiler extensions and annotations.</p>
```

Reply:
Used "behavior" since analysis and validation are not the only things you can do. In this sample, we are generating a file, like we do in docker and kubernetes modules.